messytables/excel.py

```diff
@@ -138,10 +138,51 @@ def properties(self):
 
 class XLSProperties(CoreProperties):
     KEYS = ['bold', 'size', 'italic', 'font_name', 'strikeout', 'underline',
-            'font_colour', 'background_colour', 'any_border', 'all_border']
+            'font_colour', 'background_colour', 'any_border', 'all_border',
+            'richtext']
     def __init__(self, cell):
-        self.xf = cell.sheet.book.xf_list[cell.xlrd_cell.xf_index]
-        self.font = cell.sheet.book.font_list[self.xf.font_index]
+        #self.xf = cell.sheet.book.xf_list[cell.xlrd_cell.xf_index]
+        #self.font = cell.sheet.book.font_list[self.xf.font_index]
+        self.cell = cell
+        self.merged = {}
+
+    @property
+    def xf(self):
+        return self.cell.sheet.book.xf_list[self.cell.xlrd_cell.xf_index]
+
+    @property
+    def font(self):
+        return self.cell.sheet.book.font_list[self.xf.font_index]
+
+    @property
+    def rich(self):
+        return self.cell.sheet.rich_text_runlist_map.get(self.cell.xlrd_pos, None)
```

Comment:
Should have a docstring saying what it returns, and maybe linking to https://secure.simplistix.co.uk/svn/xlrd/trunk/xlrd/doc/xlrd.html?p=4966#sheet.Sheet.rich_text_runlist_map-attribute